src/assertions/text.js

```diff
@@ -0,0 +1,12 @@
+
+import should from 'should';
+
+const Assertion = should.Assertion;
+Assertion.add('text', function(string) {
```

Comment:
Because you're using `includes` for the assert, I would rename this to `containText`. As I would expect `text` to do an exact of all the text in the node.